metals/src/main/scala/scala/meta/internal/metals/BuildServerConnection.scala

```diff
@@ -82,11 +82,6 @@ class BuildServerConnection private (
 
   def isAmmonite: Boolean = name == Ammonite.name
 
-  /* Currently only Bloop and sbt support running single test cases
-   * and ScalaCLI uses Bloop underneath.
-   */
-  def supportsTestSelection: Boolean = isBloop || isSbt || isScalaCLI
```

Comment:
It's not needed since debug is only supported in bloop (scala-cli reuses bloop) & sbt and they're both implementing test selection

Reply:
But this is still needed for other BSP servers right? Like if I'm using Mill as a BSP server, wouldn't I still want this check?

Reply:
Mill doesn't implement debug capability, in other words all build servers which provide debug implements tests selection.

Good question though, currently nothing prevents you from running for instance `metals.run-current-file` when you're using mill-bsp which will result in error delayed by 60seconds (timeout value). 

@tgodzik do you remember why is such huge timeout used? Can we reduce this to 20 seconds?

Reply:
Might have been due to tests I think. We should check if running is supported early though, why is that not possible currently?